src/newapp.ts

```diff
@@ -193,3 +200,42 @@ async function GetNewRepoName() {
     });
     return newRepoName;
 }
+
+function GetBranches(path: string) {
+    const branchPrefix = 'NVRTEMPLATE';
+    spawnSync('git', ['remote', 'add', branchPrefix, path], { env:process.env }); // addind remote

```

Comment:
Use ExecGitCommand and use some temp path to execute the commands to isolate them from any existing repo and catch the output and errors from the command.